syntaxes/dune.json

```diff
@@ -1731,6 +1731,168 @@
             }
           ]
         },
+        {
+          "comment": "melange.emit",
+          "begin": "\\([[:space:]]*(melange\\.emit)\\b",
```

Comment:
Would it make sense to recognize `melange`, `.`, and `emit` as separate entities?
Then `.` could be a `keyword.operator.dune`.

I'd be interested to know how the official dune parser reads this.

Reply:
Done, I could do coq as well in another PR